_maps/shuttles/shiptest/isv_roberts.dmm

```diff
@@ -96,25 +90,21 @@
 /turf/open/floor/plating,
 /area/ship/external)
 "hb" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer2,
 /obj/structure/cable{
 	icon_state = "1-2"
 	},
-/obj/effect/decal/cleanable/dirt/dust,
-/obj/machinery/airalarm/directional/west,
-/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer2,
 /turf/open/floor/plating,
 /area/ship/cargo)
 "hr" = (
-/obj/effect/landmark/start/assistant,
 /mob/living/simple_animal/hostile/cockroach,
-/obj/structure/safe/floor{
```

Comment:
Not sure if new players will realize where the ID is if this is removed.

Reply:
the id is both a bit of a secret, and also entirely unnecessary to begin with, its entirely impossible to get into that safe without really good ruin RNG to begin with